code/modules/projectiles/guns/projectile/automatic/zoric.dm

```diff
@@ -16,8 +16,8 @@
 	magazine_type = /obj/item/ammo_magazine/msmg
 	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_STEEL = 6, MATERIAL_PLASTIC = 4)
 	price_tag = 2000
-	damage_multiplier = 1	 // 34 lethal
-	penetration_multiplier = 0.5 // 7.5 lethal
+	damage_multiplier = 1.1	 // 34 lethal
```

Comment:
Here too.

Reply:
Should I account for 34 damage in FA or 34 damage by default, because realistically no one uses an SMG in regular fire. If you want it in FA then this is accurate.

Reply:
Actually I'll just add "in FA" to the decsription